install_opi5.sh

```diff
@@ -8,6 +8,9 @@ else
     usermod -a -G sudo pi
     mkdir /home/pi
     chown -R pi /home/pi
+
+    echo 'pi ALL=(ALL) NOPASSWD: ALL' | tee -a /etc/sudoers.d/010_pi-nopasswd >/dev/null
```

Comment:
Is there a benefit to piping to `tee -a` over using `>>` to append to this file?

Reply:
Either way is fine. I went with this approach here because it was what I validated before rebuilding the image (swapping tee for sudo tee), and the rest of the script didn't set a strong precedent one way or the other.